src/main/kotlin/no/nav/syfo/behandlendeenhet/EnhetService.kt

```diff
@@ -57,11 +57,14 @@ class EnhetService(
                 personIdentNumber = personIdent,
```

Comment:
Jeg er sikkert helt på bærtur her, men slik jeg leser denne funksjonen vil `enhetId` parameteret bety at det er sendt inn en oppfølgingsenhet. Bør vi ikke da utføre det som er inni `if`'en her om `enhetId` har verdi? 😅

Reply:
Ja, og det er vel det som skjer? (bortsett fra hvis innsendt `enhetId` samsvarer med geografisk enhet)

Reply:
Fikk det forklart av Geir 🙏 
Skriver en liten forklaring til meg selv her: Om vi eksplisitt sender inn `enhetId == null` sier vi at vi kan oppdatere personer som kan være skjermet. Dette er viktig for å kunne tilbakestille personer som har fått oppfølgingsenhet, og plutselig blir kode 6 eller kode 7.
Om `enhetId != null` ønsker vi å sjekke at personen vi prøver å sette oppfølgingsenhet for ikke er skjermet.